library/eclipse-temurin

```diff
@@ -7,32 +7,54 @@ GitFetch: refs/heads/main
 Builder: buildkit
 
 #------------------------------v8 images---------------------------------
-Tags: 8u432-b06-jdk-alpine, 8-jdk-alpine, 8-alpine
+Tags: 8u432-b06-jdk-alpine-3.20, 8-jdk-alpine-3.20, 8-alpine-3.20
 Architectures: amd64
-GitCommit: d7a5038edcd8ab08b0babaeae09d0c097453a023
-Directory: 8/jdk/alpine
+GitCommit: 223947cdea5cc88bdeecb26e600e215bc785b1cc
+Directory: 8/jdk/alpine/3.20
+
+Tags: 8u432-b06-jdk-alpine-3.21, 8-jdk-alpine-3.21, 8-alpine-3.21
+SharedTags: 8u432-b06-jdk-alpine, 8-jdk-alpine, 8-alpine
```

Comment:
`SharedTags` is designed to provide a way for multiple Windows images (one per `os.version`) plus up to one Linux image to be pullable by the same name. I don't see any of the `*-alpine` tags as being useful to that goal (i.e., there would not be a Windows-based `8-alpine` image). Can we move these `*-alpine` SharedTags into the respective `Tags` fields?

Reply:
I've fixed this now